addon-test-support/pages/-private/ember-table-header.js

```diff
@@ -131,14 +139,14 @@ export default {
   scope: 'thead',
 
   /**
-   * List of columns in the header.
+   * Selects all non-slack `th` header elements from all header rows.
    */
   headers: collection('th:not([data-test-ember-table-slack])', Header),
 
   /**
-   * List of columns in the header, excluding slack column if present.
+   * Selects all slack `th` header elements from all header rows.
    */
-  contentHeaders: collection('th:not([data-test-ember-table-slack])', Header),
+  slackHeaders: collection('th[data-test-ember-table-slack]', Header),
```

Comment:
I am assuming this private API. Is that correct?

Reply:
Good question. Moved "offline" for a broader discussion.

In this case, `contentHeaders` is redundant, unused internally, and new since 3.0. It's probably safe to assume nobody is using it. More clarity would be good here though.